Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -907,11 +907,15 @@ def drilldown_enrichment(service, notable_data, num_enrichment_events):
         if searchable_query:
             status, earliest_offset, latest_offset = get_drilldown_timeframe(notable_data, raw_dict)
             if status:
-                if "latest=" not in searchable_query:
-                    searchable_query = "latest={} ".format(latest_offset) + searchable_query
-                if "earliest=" not in searchable_query:
-                    searchable_query = "earliest={} ".format(earliest_offset) + searchable_query
                 kwargs = {"count": num_enrichment_events, "exec_mode": "normal"}
+                if latest_offset:
+                    kwargs['latest_time'] = latest_offset
+                    if "latest=" not in searchable_query:
+                        searchable_query = "latest={} ".format(latest_offset) + searchable_query
```

Comment:
do we need `latest` and `earliest` if we send them  in the request `latest_time` and `earliest_time`